### PR TITLE
[New Rule] Persistence via Docker Shortcut Modification

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/732

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/macos/persistence_docker_shortcuts_plist_modification.toml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2020/12/18"
+maturity = "production"
+updated_date = "2020/12/18"
+
+[rule]
+author = ["Elastic"]
+description = """
+An adversary can establish persistence by modifying an existing macOS Docker property list in order to execute a
```
Comment (brokensound77) [inline rules/macos/persistence_docker_shortcuts_plist_modification.toml:9]:
```suggestion
An adversary can establish persistence by modifying an existing macOS dock property list in order to execute a
```

File: rules/macos/persistence_docker_shortcuts_plist_modification.toml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+]
+risk_score = 47
+rule_id = "c81cefcb-82b9-4408-a533-3c3df549e62d"
+severity = "medium"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Persistence"]
+type = "eql"
+
+query = '''
+file where event.type == "change" and
+ wildcard(file.path, "/Users/*/Library/Preferences/com.apple.dock.plist") and not process.name in ("xpcproxy", "cfprefsd")
```
Comment (brokensound77) [inline rules/macos/persistence_docker_shortcuts_plist_modification.toml:28]: nit:

```suggestion
file where event.type == "change" and
  wildcard(file.path, "/Users/*/Library/Preferences/com.apple.dock.plist") and 
  not process.name in ("xpcproxy", "cfprefsd")
```

**Review by brokensound77 — APPROVED** [review]
A couple comments, but LGTM 👍

**Review by threat-punter — APPROVED** [review]
😎

**Pull request merged**